drivers/ethernet/Kconfig.mcux

```diff
@@ -75,6 +75,13 @@ config ETH_MCUX_0_RANDOM_MAC
 config ETH_MCUX_0_MANUAL_MAC
 	bool "Manual MAC address"
 
+config ETH_MCUX_HW_ACCELERATION
+	bool "Enable hardware"
+	default y
+	help
+		Enabled hardware acceleration for the following:
```

Comment:
Help text indent - should be 2 spaces, looking at the context around.

"Enabled" - should be "Enable" ?